src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -15,7 +15,8 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response)
         throws IOException, ServletException{
-        response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
+        request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
+        //response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
     }
 
```

Comment:
Do you need a doPost method?  How is the registration actually happening?  (Like, when a user fills in a username and a password, how does it get saved to the database?  Is that coming in a separate PR?